src/Logary/DataModel.fs

```diff
@@ -275,25 +275,8 @@ and SpanInfo =
   static member formatId (id: Guid) =
     id.ToString("n")
 
-/// Why was the message/metric/event not logged?
-[<Struct>]
-type LogError =
```

Comment:
Thing is, if we change this, there's a whole heap of work to do in the Facade as well. I'm OK with having this **very** simple, like now, because we have only a few cases that happen:

 - target fails terminally due to target's state (e.g. out of disk)
 - target fails grey due to e.g. due to too slow processing 
 - connectivity broken / DNS lookup failure, etc; all inside the logary target, and those may be cached, so a service restart is needed
 - rate of logging > rate of sending, dropping in part
 - misconfiguration

We don't really want to have timeouts in the pipeline if we can avoid it, because every time we add another timeout, it's a work item that Hopac's scheduler has to track. If the Hopac scheduler's queue can't keep up, it means the buffers will fill up, which means we'll run out of memory and crash due to that (since we still input messages into the buffer pipeline); that was the reason of the refactor this summer, away from 'log simple' having a timeout alternative, towards this.

Reply:
there is a litter difference here, in old one: message will not be dropped when the timeout alternative committed, since we have more than one targets. but right now, if timeout, then message will not be append to the buffer, and we will get a timeout error message to return to the client. if we config a global logging error handler on the registry, we can notify this situation, then adjust the target buffer length ,or speed up the processing. 

since we have a timeout protocol, user can do backpressure with some balance, no need to wait all the time (since logging is not such important as the business code), but we still want to give some time to buffer if they can't catch up in a short time.